### PR TITLE
Added Switzerland subdivisions

### PR DESCRIPTION
**Pull request by JackArrr**
The list of subdivisions of Switzerland based on ISO 3166-2:CH standard.
Source: https://en.wikipedia.org/wiki/ISO_3166-2:CH ISO 3166-2:CH

### CODE REVIEW

File: modules/ch-subdivisions.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * The list of subdivisions of Uruguay based on ISO 3166-2:CH standard.
```
Comment (takayukister) [inline modules/ch-subdivisions.php]: Uruguay -> Switzerland

File: modules/ch-subdivisions.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ *
+ * Source: https://en.wikipedia.org/wiki/ISO_3166-2:CH ISO 3166-2:CH
+ */
+class Listo_CH_Subdivisions implements Listo {
+	private function __construct() {}
+
+	public static function items() {
+		return array(
+			'AG' => _x( 'Aargau','ch_subdivisions','listo' ),
```
Comment (takayukister) [inline modules/ch-subdivisions.php]: Use lowercase in array keys

File: modules/ch-subdivisions.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+			'AI' => _x( 'Appenzell Innerrhoden','ch_subdivisions','listo' ),
+			'AR' => _x( 'Appenzell Ausserrhoden','ch_subdivisions','listo' ),
+			'BE' => _x( 'Bern','ch_subdivisions','listo' ),
+			'BL' => _x( 'Basel-Land','ch_subdivisions','listo' ),
```
Comment (takayukister) [inline modules/ch-subdivisions.php]: Use the name that is used in the source data ([https://en.wikipedia.org/wiki/ISO_3166-2:CH](https://en.wikipedia.org/wiki/ISO_3166-2:CH))

**Review by takayukister — CHANGES_REQUESTED** [review]

File: modules/ch-subdivisions.php (hunk 3 above)
Comment (JackArrr) [reply modules/ch-subdivisions.php]: Sorry, should now be perfect fitted and sorted.

**Comment by JackArrr** [comment]
Fixed the issues.

File: modules/ch-subdivisions.php [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ * The list of subdivisions of Switzerland based on ISO 3166-2:CH standard.
+ *
+ * Source: https://en.wikipedia.org/wiki/ISO_3166-2:CH ISO 3166-2:CH
+ */
+class Listo_CH_Subdivisions implements Listo {
+	private function __construct() {}
+
+	public static function items() {
+		return array(
+			'ag' => _x( 'Aargau','ch_subdivisions','listo' ),
```
Comment (takayukister) [inline modules/ch-subdivisions.php]: Put a space after a comma

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage

**Review by takayukister — CHANGES_REQUESTED** [review]

**Comment by JackArrr** [comment]
Putted spaces after comma to meet wordpress coding standards.

**Comment by takayukister** [comment]
Thanks!

**Pull request merged**